tests/test_random_state.py

```diff
@@ -473,10 +473,9 @@ def test_distr(self, usm_type):
         data = RandomState(seed, sycl_queue=sycl_queue).randn(3, 2, usm_type=usm_type)
         assert_array_almost_equal(data.asnumpy(), desired, decimal=precision)
 
-        # TODO: discuss with oneMKL: return 0.0 instead of the 1st element
         # call with omitted dimensions has to draw the first element from desired
-        # actual = dpnp.asnumpy(RandomState(seed).randn(usm_type=usm_type))
-        # assert_array_almost_equal(actual, desired[0, 0], decimal=precision)
+        actual = dpnp.asnumpy(RandomState(seed).randn(usm_type=usm_type))
+        assert_array_almost_equal(actual, desired[0, 0], decimal=precision)
```

Comment:
It should probably check that repeated scalar draws replicate the sequence hard-coded in the `desired` array. May I suggest to rename that variable to `expected`.